Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_include.yaml

```diff
@@ -0,0 +1,80 @@
+
+id: f7abe9c1-1e6c-4317-b907-25769e7764c5
```

Comment:
There should not be 2 templates with the same ID, but the id f7abe9c1-1e6c-4317-b907-25769e7764c5 is duplicated. Please check and update, thanks.